### PR TITLE
Adding my package Geomcompare

### PR DESCRIPTION
**Pull request by MTachon**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MTachon** [comment]
I am the only author of the package and maintainer of this recipe. It is fine for me to be listed here.

File: recipes/geomcompare/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7e88099f531f3d8dfd293eb1ed29121ce70b214d2581ad6b33ce33b75200fdc9
+  md5: 99115b09a972b0c53babbe074cdd0747
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:11]:
```suggestion
```

File: recipes/geomcompare/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  number: 0
+  skip: True  # [py < 39 or osx]
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:15]:
```suggestion
  [skip: True  # [py < 39 or osx]](noarch: python)
```

File: recipes/geomcompare/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:20]:
```suggestion
    - python >=3.9
```

File: recipes/geomcompare/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - pip
+  run:
+    - python
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:23]:
```suggestion
    - python >=3.9
```

File: recipes/geomcompare/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    - numpy
+    - shapely
+    - psycopg2
+    - rtree
+    - pyproj
+    - gdal
+    - libspatialite
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:30]: These are not defined upstream. Are they extras?

File: recipes/geomcompare/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+test:
+  imports:
+    - geomcompare
```
Comment (ocefpaf) [inline recipes/geomcompare/meta.yaml:34]:
```suggestion
    - geomcompare
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/geomcompare/LICENSE.rst [hunk 7]
```diff
@@ -0,0 +1,562 @@
+GNU General Public License
```
Comment (ocefpaf) [inline recipes/geomcompare/LICENSE.rst:1]: The license is in the package. You don't need this file here.

**Review by ocefpaf — CHANGES_REQUESTED** [review]

**Pull request closed without merging**